kod/object/active/holder/nomoveon/battler/player/user.kod

```diff
@@ -3749,6 +3758,41 @@ messages:
       return;
    }
 
+   % Server has received a client request to move an item from the first item's position
+   %   in the inventory list to a second item's position.  This function executes the move
+   %   in the plPassive inventory list assuming all checks clear.
+   UserMoveInventoryItem(payload_index=1,target_index=1)
+   {
+      local plPassive_max_index;
+
+      if payload_index = $
+         OR target_index = $
+         OR payload_index = target_index
+         OR plPassive = $
+      {
+         Debug("ALERT! ",Send(self,@GetTrueName),self,"tried to move "
+                  "an object or an object to a target in their inventory that does not exist!");
+         return;
+      }
+
+      % Check to make sure payload_index and target_index are within plPassive
```

Comment:
This comment is not quite right; the indexes are allowed to be 1 past the end of plPassive to indicate moving to the end of the list.

Reply:
Changed comment slightly as requested.